packages/backend-core/src/security/roles.ts

```diff
@@ -50,7 +51,7 @@ export class Role implements RoleDoc {
   _id: string
   _rev?: string
   name: string
-  permissionId: string
+  permissionId: BuiltinPermissionID
```

Comment:
Will this not be used for custom roles, as well?

Reply:
Yes, this is used in all roles, but its never a `string` - its just a very old type, we've always validated the Role to make sure it is one of the values in `BuiltinPermissionID`.